.github/workflows/ci.yml

```
@@ -2,24 +2,22 @@ name: CI
+          node-version: '20'
+          cache: 'npm'
       - run: npm ci
-      - name: Lint
-        run: npm run lint
-      - name: Build
-        run: npm run build
-      - name: Docs (TypeDoc)
-        run: npm run docs:api
+      - run: npm run lint
       - run: npm test -- --run
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)